style/styles_products.css

```diff
@@ -0,0 +1,257 @@
+.products {
+  padding-top: 54px;
+}
+
+.products__title {
+  margin: 0;
+  padding-bottom: 16px;
+  text-align: center;
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 22px;
+}
+
+.products__block {
+  display: flex;
+  justify-content: space-between;
+}
+
+.products-shop {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+}
+
+.products-shop__pic {
+  display: inline-block;
+  padding-left: 23px;
+  padding-right: 19px;
+  padding-bottom: 30px;
+}
+
+.products-shop__model {
+  display: inline-block;
+  font-family: "Gotham Pro Light", sans-serif;
+  font-size: 12px;
+  padding-left: 12px;
+  padding-bottom: 7px;
+}
+
+.products-shop__block {
+  display: flex;
+  justify-content: space-between;
+}
+
+.products-shop__price {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 14px;
+  padding-left: 12px;
+}
+
+.products-shop__block-ellipse {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 3px;
+}
+
+.products-shop__ellipse {
+  border-radius: 50%;
+  height: 7px;
+  width: 7px;
+  margin-right: 8px;
+}
+
+.ellipse_gray {
+  background-color: #c4c4c4;
+}
+
+.ellipse_black {
+  background-color: #0E1211;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#0E1211" to be "#0e1211" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5525083)